idaes/models/flowsheets/methanol_flowsheet.py

```diff
@@ -0,0 +1,736 @@
+import idaes.core.util.unit_costing as costing
+
+
+def build_model(m):
+    # Define model components and blocks
+    m.fs = FlowsheetBlock(default={"dynamic": False})
+
+    m.fs.thermo_params_VLE = GenericParameterBlock(
+        default=thermo_props_VLE.config_dict)
+
+    m.fs.thermo_params_VLE.set_default_scaling("flow_mol", 1)
+    m.fs.thermo_params_VLE.set_default_scaling("temperature", 1e-2)
+    m.fs.thermo_params_VLE.set_default_scaling("pressure", 1e-2)
+    m.fs.thermo_params_VLE.set_default_scaling("enth_mol", 1e-3)
+    m.fs.thermo_params_VLE.set_default_scaling("entr_mol", 1e-1)
+    for comp in thermo_props_VLE.config_dict["components"]:
+        m.fs.thermo_params_VLE.set_default_scaling("mole_frac_comp",
+                                                   1e2, index=comp)
+        m.fs.thermo_params_VLE.set_default_scaling("enth_mol_comp",
+                                                   1e2, index=comp)
+        m.fs.thermo_params_VLE.set_default_scaling("entr_mol_phase_comp",
+                                                   1e2, index=comp)
+        for attr in dir(getattr(m.fs.thermo_params_VLE, comp)):
+            if 'coef' in attr:
+                iscale.set_scaling_factor(getattr(
+                    getattr(m.fs.thermo_params_VLE, comp), attr), 1)
+        m.fs.thermo_params_VLE.set_default_scaling("flow_mol_phase_comp",
+                                                   1, index=comp)
+
+    m.fs.thermo_params_vapor = GenericParameterBlock(
+        default=thermo_props_vapor.config_dict)
+
+    m.fs.thermo_params_vapor.set_default_scaling("flow_mol", 1)
+    m.fs.thermo_params_vapor.set_default_scaling("temperature", 1e-2)
+    m.fs.thermo_params_vapor.set_default_scaling("pressure", 1e-2)
+    m.fs.thermo_params_vapor.set_default_scaling("enth_mol", 1e-3)
+    m.fs.thermo_params_vapor.set_default_scaling("entr_mol", 1e-1)
+    for comp in thermo_props_vapor.config_dict["components"]:
+        m.fs.thermo_params_vapor.set_default_scaling("mole_frac_comp",
+                                                     1e2, index=comp)
+        m.fs.thermo_params_vapor.set_default_scaling("enth_mol_comp",
+                                                     1e2, index=comp)
+        m.fs.thermo_params_vapor.set_default_scaling("entr_mol_phase_comp",
+                                                     1e2, index=comp)
+        for attr in dir(getattr(m.fs.thermo_params_vapor, comp)):
+            if 'coef' in attr:
+                iscale.set_scaling_factor(getattr(
+                    getattr(m.fs.thermo_params_vapor, comp), attr), 1)
+        m.fs.thermo_params_vapor.set_default_scaling("flow_mol_phase_comp",
+                                                     1, index=comp)
+
+    m.fs.reaction_params = GenericReactionParameterBlock(
+        default={"property_package": m.fs.thermo_params_vapor,
+                 **reaction_props.config_dict})
+
+    # mixing feed streams
```

Comment:
If this is going to be a proper example, then you need to use `Feed` blocks to to define the inlets and connect those to the mixer.